eth/stagedsync/stage_headers.go

```diff
@@ -173,14 +181,14 @@ func HeadersPOS(
 		return err
 	}
 	if parent != nil && parent.Hash() == header.ParentHash {
-		if err := cfg.hd.VerifyHeader(header); err != nil {
+		/*if err := cfg.hd.VerifyHeader(header); err != nil {
```

Comment:
What was the problem with header verification? We should fix it, not disable it.